### ISSUE TITLE
[REGRESSION]:After Update playwright from 1.31.2 -> 1.32.0 my test.ts  with fixtures is compiling with other tests nested in folder

### ISSUE THREAD

**Issue by ChrystianC** [microsoft/playwright#22360]
**Context:**
- GOOD Playwright Version: 1.31.2
- BAD Playwright Version: 1.32.0
- Operating System: Linux

After update playwright. My test.ts(name of file) with fixtures is compiling with other test nested in other folder.

```
Error: test file "tests/foo/save.test.ts" should not import test file "test.ts"
```

**Describe the bug**
Before version 1.32.0 works fine

**Comment by ranma42** [comment]
Might be a duplicate of https://github.com/microsoft/playwright/issues/22114

**Comment by mxschmitt** [comment]
Also filed here by Kent: https://discord.com/channels/807756831384403968/1088481230800035890/1088481230800035890

**Comment by dgozman** [comment]
Fixed by #22440.
